### PR TITLE
Use ableton.pkg_mgr_path paths

### PR DESCRIPTION
**Pull request by nre-ableton**
- Introduce dependency on ableton.pkg_mgr_path
- Use ableton.pkg_mgr_path paths

### CODE REVIEW

**Review by ala-ableton — APPROVED** [review]

**Review by ala-ableton — CHANGES_REQUESTED** [review]
Actually, this would change would be worth a version bump.

File: Jenkinsfile [hunk 1]
```diff
@@ -7,6 +7,9 @@ devToolsProject.run(
   setup: { data ->
     Object venv = virtualenv.create('python3.8')
     venv.run('pip install -r requirements-dev.txt')
+    sshagent(['build-ssh-key']) {
```
Comment (ala-ableton) [inline Jenkinsfile:10]: Shouldn't we instead use HTTPS to get `Ableton/ansible-role-pkg-mgr-path`?

**Review by ala-ableton — COMMENTED** [review]
Please rebase/squash, and I'll have another look, thanks!

**Comment by nre-ableton** [comment]
@ablbot autosquash

**Review by ala-ableton — APPROVED** [review]

**Pull request merged**